packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -276,6 +274,13 @@ class Splitter extends CollectionWidget<Properties> {
     return index === findLastIndexOfVisibleItem(items);
   }
 
+  _fireContentReadyAction(): void {
+    // @ts-expect-error badly typed base class
```

Comment:
We shouldn't leave the // @ts-expect-error comment. You can type this method inside edit.ts.